test/built-ins/Array/prototype/splice/create-species-neg-zero.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.26
+esid: sec-array.prototype.splice
+description: The value `-0` is converted to `0`
+info: |
+    [...]
+    3. Let relativeStart be ? ToInteger(start).
+    4. If relativeStart < 0, let actualStart be max((len + relativeStart), 0);
+       else let actualStart be min(relativeStart, len).
+    5. If the number of actual arguments is 0, then
+       [...]
+    6. Else if the number of actual arguments is 1, then
+       [...]
+    7. Else,
+       a. Let insertCount be the number of actual arguments minus 2.
+       b. Let dc be ? ToInteger(deleteCount).
+       c. Let actualDeleteCount be min(max(dc, 0), len - actualStart).
```

Comment:
This is just cosmetic, but it is cloudy for me. I believe `actualDeleteCount` becomes `+0` here on the max operation. In any way this is not observable before the step 2 of 9.4.2.3.